src/components/ScholarshipAmountField.jsx

```diff
@@ -14,7 +14,7 @@ import AmountTextField from './AmountTextField';
 
 const useStyles = makeStyles((theme) => ({
   formControlStyle: {
-    paddingTop: theme.spacing(2),
+    paddingBottom: theme.spacing(1),
   },
```

Comment:
Rather than hardcode style the AmountTextField I suggest receiving the classname with the styles in the parent. Note that this is for the overall formcontrol and not the inner amount textfields